REVIEWING.md

```diff
@@ -0,0 +1,29 @@
+# Reviewing and Merging Pull Requests
+
+This document is a guideline for how core contributors should review and merge
+pull requests (PRs).
+
+PRs may only be merged after the following criteria are met:
+
+1. It has been open for at least 1 business day
+1. It has no `veto` label on it
+1. It has been 'LGTM'-ed by 3 different reviewers
+
+## LGTMs
+
+When a reviewer deems a PR ready for review, they should add a comment to the PR
```

Comment:
I don't think the phrase is "ready for review", I think you mean "ready for merge" or something like that.
And in the next line too.